.github/workflows/ci.yml

```diff
@@ -315,6 +360,107 @@ jobs:
       # I don't want to build SID just for sake of its -l test
       run: ${{ matrix.make }} -r -j $((${{ steps.cpu-cores.outputs.count }} + 1)) -C ${{ env.wc }} BUILD=../${{ env.build }} ${{ matrix.san }}=1 ${{ matrix.debug }}=1 PKGCONF=pkg-config SID='true; echo sid' CC=${{ matrix.cc }} NODOC=1 LX=../${{ env.build }}/bin/lx test
 
+  test_fuzz:
+    name: "Fuzz (mode ${{ matrix.mode }}) ${{ matrix.cc }} ${{ matrix.os }} ${{ matrix.debug }}"
+    runs-on: ${{ matrix.os }}-latest
+    timeout-minutes: 5 # this should never be reached, it's a safeguard for bugs in the fuzzer itself
+    needs: [ build ]
+
+    strategy:
+      fail-fast: false
+      matrix:
+        san: [ FUZZER ]
+        os: [ ubuntu ]
+        cc: [ clang ]
+        make: [ bmake ]
+        debug: [ DEBUG, RELEASE ] # RELEASE=1 is a no-op
+        mode: [ m, p, d ]
```

Comment:
Are the seeds from these modes cached distinctly? Failing seeds from one mode are likely to pass in another, but that shouldn't be treated as a sign that the root cause of the failure has been addressed.

Reply:
yes, the mode is part of the cache key. look for `key:`

Reply:
this part:
```
    - name: Save seeds (mode ${{ matrix.mode }}-${{ matrix.debug }})
      uses: actions/cache/save@v3
      if: always()
      with:
        path: ${{ env.seeds }}-${{ matrix.mode }}
        key: ${{ steps.cache-seeds.outputs.cache-primary-key }}
```